shop.go

```diff
@@ -34,7 +35,7 @@ type Shop struct {
 */
 func NewShop(shop string, accessToken string) Shop {
 	return NewCustomShop(
-		fmt.Sprintf("https://%v.myshopify.com/admin/api/2021-04", shop),
+		fmt.Sprintf("https://%v.myshopify.com/admin/api/2022-07", shop),
```

Comment:
Did you check everything else still worked for this? I always imagined this would be a nightmare.

Reply:
I haven't checked everything else still works. Anything using `httpshopify` right now is currently being served `2022-01` implicitly as the minimum version. So I'm tempted to say just checking through breaking changes between `2022-01` and `2022-07` will suffice?

Reply:
Yeh it should be if Shopify have done the versioning right which you would expect them to.

Reply:
Cool, the only other removals other than this, is [this one about `CustomerSavedSearch`s](https://shopify.dev/docs/api/release-notes/2022-04#rest-admin-api-changes). Can't find any usage here so think we're good on this.

Reply:
Yeh not used that one I don't think 👍